daemon/volumes.go

```diff
@@ -366,3 +381,12 @@ func (container *Container) unmountVolumes() {
 		}
 	}
 }
+
+// unmountVolumesOnError is a convenient deferrable method which will unmount
```

Comment:
Any reason this isn't near the other one up above?  Just to keep util funcs together.